examples/make_titanic_example_data.py

```diff
@@ -92,6 +99,13 @@ def main():
     test_fs = Reader.for_path('test.csv',
```

Comment:
It looks like `test.csv` does not have a `Survived` column:
```
➜  titanic git:(streamline-objectives) ✗ head test.csv
PassengerId,Pclass,Name,Sex,Age,SibSp,Parch,Ticket,Fare,Cabin,Embarked
892,3,"Kelly, Mr. James",male,34.5,0,0,330911,7.8292,,Q
```

This is the error I get:
```
(/Users/mmulholland/Documents/skll_dev/skll_env) ➜  titanic git:(streamline-objectives) ✗ ../../skll_env/bin/python ../make_titanic_example_data.py
2019-10-14 13:05:28,426 - skll.data.readers - INFO - Rows/lines with any blank values will be dropped.
Loading train.csv...           done
Writing titanic/train/vitals.csv...done
Writing titanic/train/socioeconomic.csv...done
Writing titanic/train/family.csv...done
Writing titanic/train/misc.csv...done
Writing titanic/train+dev/vitals.csv...done
Writing titanic/train+dev/socioeconomic.csv...done
Writing titanic/train+dev/family.csv...done
Writing titanic/train+dev/misc.csv...done
Writing titanic/dev/vitals.csv...done
Writing titanic/dev/socioeconomic.csv...done
Writing titanic/dev/family.csv...done
Writing titanic/dev/misc.csv...done
Traceback (most recent call last):
  File "../make_titanic_example_data.py", line 124, in <module>
    main()
  File "../make_titanic_example_data.py", line 109, in main
    sparse=False).read()
  File "/Users/mmulholland/Documents/skll_dev/skll/data/readers.py", line 427, in read
    ids, labels, features = self._sub_read(self.path_or_list)
  File "/Users/mmulholland/Documents/skll_dev/skll/data/readers.py", line 844, in _sub_read
    df = pd.read_csv(file, sep=self._sep, engine=self._engine, **self._pandas_kwargs)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 685, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 457, in _read
    parser = TextFileReader(fp_or_buf, **kwds)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1135, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1959, in __init__
    _validate_usecols_names(usecols, self.orig_names)
  File "/Users/mmulholland/Documents/skll_dev/skll_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1255, in _validate_usecols_names
    "columns expected but not found: {missing}".format(missing=missing)
ValueError: Usecols do not match columns, columns expected but not found: ['Survived']
```

Reply:
Good catch! Will fix.